python/desc/sims/GCRCatSimInterface/HostImages.py

```diff
@@ -0,0 +1,100 @@
+import os
+import numpy as np
+import pandas as pd
+from lsst.sims.utils import angularSeparation
+
+__all__ = ['hostImage']
+
+class hostImage(object):
```

Comment:
Please include a docstring indicating what this class is meant to be used for.

Also include definitions of the parameters we should feed to the class' `__init__` method.